### ISSUE TITLE
A11y_.NET Core_WinForms_PrintingTesting_Print_ColorContrast: Color contrast ratio of the text "Stores printing output to a file" which is present in the "Print" Screen ratio is 3.278:1 which is less than 4.5:1.

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9198]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#DesktopApp;#Win11;#NonBenchmark;#WCAG1.4.3;#ColorContrast

### Actual:
Color contrast ratio of the text present in the "Print" screen i.e., "Stores printing output to a file" ratio is 3.278:1 which is less than 4.5:1.

### Expected:
Color contrast ratio of the text present in the "Print" screen i.e., "Stores printing output to a file" ratio should be greater than or equal to the 4.5:1.

### User Impact:
Low vision users won't be able to see the text present on the screen i.e., "Stores printing output to a file".

### Attachments:
![Color Contrast](https://github.com/dotnet/winforms/assets/125462007/4f57cce0-580a-4f3f-bcc8-a554d97a9fcd)

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Click on Printing_Controls:Testing the controls under Printing Tab.
4. Navigate to Print button and click on it.
5. Print Screen would appear.
6. Now TAB to the text present below "Print to File" toggle button.
7. Open Accessibility Insights for Windows.
8. Observe that color contrast ratio of the text present in the "Print" screen i.e., "Stores printing output to a file" ratio should be greater than or equal to the 4.5:1.

**Comment by elachlan** [comment]
@dmitrii-drobotov would your team look at this?

**Comment by 12-shweta** [comment]
#BM_.NET Core[WPF]_Win32_May23;

**Comment by MandiMan** [comment]
This issue is not a regression problem, can be reproduced on .NET 3.1 & 5.0 & 6.0 & 7.0 & 8.0 and .NET Framework apps.
![image](https://github.com/dotnet/winforms/assets/108860782/9b512f67-c992-40e5-ac04-8efa98316348)

**Comment by elachlan** [comment]
Is this a winforms issue or a windows issue? It looks like a windows based dialog, not winforms?

**Comment by HindujaArugula** [comment]
@elachlan This comes under Winforms & it is not a windows-based dialog.

**Comment by merriemcgaw** [comment]
I'll see if we can add this to the general Common Dialogs problem

**Comment by merriemcgaw** [comment]
This is on Windows Side and the [C&AI Exception ](https://ceacc.visualstudio.com/ceaccex/_workitems/edit/165844/ )has been approved, I'd like to close this issue out. @HindujaArugula can you please close this?

**Comment by PoojaNamde** [comment]
#A11ypreview; #A11yExceptionApproved; #Regressed:09-13-23;

Added A11ypreview tag and A11yExceptionApproved tag.

@merriemcgaw 
Here we cannot close the bug as exception is approved. Added the tags accordingly.

**Comment by merriemcgaw** [comment]
@PoojaNamde Is there anything you need to add to indicate that this is External to WinForms and we have zero control. This is the same dialog that pops when you try to print from notepad for example. Also, what is the value of adding the preview tag as this is present in Windows11 and therefore 100% of .NET and .NET Framework versions would result in the same dialog.

It also doesn't make sense that the issue needs to be active in our repo and regressed every so often. If we remove the button that calls the print dialog, would that be sufficient to close the issue and avoid your team having to continuously regress something that really isn't our product after all.

**Comment by merriemcgaw** [comment]
After speaking with Padmaja, we will close the GitHub issue, but will won't have the closed tag added.

**Comment by HindujaArugula** [comment]
#Regressed:09-27-23;
